api/src/middleware/respond.ts

```diff
@@ -67,6 +68,22 @@ export const respond: RequestHandler = asyncHandler(async (req, res) => {
 				return stream.pipe(json2csv).pipe(res);
 			}
 		}
+		if (['xliff', 'xliff2'].includes(req.sanitizedQuery.export)) {
```

Comment:
I think we'll end up moving this whole logic (including json/csv) into a dedicated ExportService (as part of https://github.com/directus/directus/issues/2927)

Reply:
Do you want me to work on that in scope of this task? Or you want me to finish first with that one and then start to work on https://github.com/directus/directus/issues/2927? I, honestly, not sure if I will have enough time to work on big tasks within the upcoming month, but maybe later...

Reply:
No that's ok! I'll pick up the work on #2927 and get that across the finish line. Then we can update this PR to fit within the finalized structure for that 🙂

Reply:
Ok